src/core/client/admin/components/BanModal.css

```diff
@@ -15,3 +15,34 @@ $ban-modal-text: var(--palette-text-500);
 .sitesToggle {
   margin-bottom: var(--spacing-3);
 }
+
+.domainBanHeader {
+  margin-bottom: var(--spacing-2);
+
```

Comment:
Can you clean up this extra space here?

Reply:
✅

Reply:
I'm still seeing an extra space here?

Reply:
ah, i only noticed the extra space within the brackets

Reply:
✅